test/recipes/75-test_quicapi.t

```diff
@@ -35,6 +35,12 @@ ok(run(test(["quicapitest", "default",
              srctop_dir("test", "recipes", "75-test_quicapi_data")])),
              "running quicapitest");
 
+ok(run(test(["quicapitest", "default",
+             srctop_file("test", "default-recordpadding.cnf"),
+             srctop_dir("test", "certs"),
+             srctop_dir("test", "recipes", "75-test_quicapi_data")])),
+             "running quicapitest");
```

Comment:
This runs the entire quicapi test again, which currently on master has 33 sub-tests (and some of those with numerous iterations). That seems excessive for what we are trying to achieve here. We should seek to avoid an excessively long execution time for our test runs. We should instead aim to run some much quicker binary which does the minimum necessary to test what we are doing.

Reply:
The quicapi test runs locally in 1.06 seconds of wall clock time,  so I thought the tradeoff between creating a whole new test suite and just running the same test with a config that covered our new test case within the existing tests was biased in favor of the latter.  I can add a new suite if thats the consensus, but it seems to me like alot of extra code for a test that can be covered by what we already have.